main.py

```diff
@@ -1,24 +1,27 @@
-#---------------------UI Server, Connects all Components-------------------------#
+"""UI Server, Connects all Components."""
 
-from flask import Flask, jsonify, request, json, render_template, send_file, make_response, abort
-from flask_restful import Api, Resource, reqparse
-from restAPI import ReleaseList, Release, Pagination, GetLabels, GetTasks
+from flask import Flask
+from flask import make_response
+from flask_restful import Api
+from rest_api import RestAPI
 
 
 # creating the Flask application
-app = Flask(__name__)
-api = Api(app)
+APP = Flask(__name__)
+API = Api(APP)
 
-api.add_resource(ReleaseList, '/releases')
-api.add_resource(Release, '/release')
-api.add_resource(Pagination, '/page')
-api.add_resource(GetLabels, '/labels')
-api.add_resource(GetTasks, '/tasks')
+REST = RestAPI()
+
```

Comment:
I am thinking that using a swagger definition might be better here. We could do something like GET releases/release-0.1 or POST releases/release-1.0.0
http://michal.karzynski.pl/blog/2016/06/19/building-beautiful-restful-apis-using-flask-swagger-ui-flask-restplus/

Reply:
let s talk about this.